### DISCUSSION TITLE
Helix vs Neovim in regard to zk-vim

### DISCUSSION THREAD

**Discussion by gour** [zk-org/zk#260] [Help] [answered]
Hello,

I'd like to stay with modal editor instead of using Emacs and its note-taking/Zettelkasten packages (org-roam, denote,...). For some time I was considering to use [nb](https://github.com/xwmx/nb), but it uses 3rd party plugin for backlinking and **no** graphing capabilities which are  zk's "pros". :smile: 

However, I'd like to spend more time doing actual note-taking instead of fiddling with my tools, so considering to use [Helix](https://github.com/helix-editor/helix) editor which requires (much) less setup than neovim, so wonder how much I'd lose by not using e.g. `zk-vim` but just LSP-capable editor?

**Comment by zalegrala** [comment, marked as answer]
You'd probably lose access to the Lua api, which is handy for making key bindings.

**Comment by mickael-menu** [comment]
Using any LSP client you have access to these features:

https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#language-server-protocol

`zk-nvim` brings some additional UI features such as a note or tag picker and convenient Lua APIs wrapping the `zk` LSP commands: https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#custom-commands
